src/main/java/com/artipie/Pie.java

```diff
@@ -67,20 +70,24 @@ public Pie(final Settings settings) {
     @SuppressWarnings("PMD.OnlyOneReturn")
     public Response response(final String line, final Iterable<Map.Entry<String, String>> headers,
         final Publisher<ByteBuffer> body) {
+        Logger.info(this, "Request: %s", line);
         final URI uri = new RequestLineFrom(line).uri();
         if (uri.getPath().equals("*")) {
             return new RsWithStatus(200);
         }
-        final String[] path = uri.getPath().split("/");
+        final String[] path = uri.getPath().replaceAll("^/+", "").split("/");
```

Comment:
@g4s8 Would be smarter to assign `uri.getPath()` to a variable once after line 74  and use it.
This method `response` defines `uri.getPath()` twice.

Reply:
@fanifieiev right, fixed